src/hardware/ston1_dac.cpp

```diff
@@ -0,0 +1,86 @@
+/*
+ *  Copyright (C) 2022-2022  The DOSBox Staging Team
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along
+ *  with this program; if not, write to the Free Software Foundation, Inc.,
+ *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+#include "ston1_dac.h"
+
+#include <cassert>
+
+#include "checks.h"
+
+CHECK_NARROWING();
+void StereoOn1::BindToPort(const io_port_t lpt_port)
+{
+	using namespace std::placeholders;
+	const auto write_data = std::bind(&StereoOn1::WriteData, this, _1, _2, _3);
+	const auto read_status = std::bind(&StereoOn1::ReadStatus, this, _1, _2);
+	const auto write_control = std::bind(&StereoOn1::WriteControl, this, _1, _2, _3);
+	BindHandlers(lpt_port, write_data, read_status, write_control);
+	LOG_MSG("LPT_DAC: Initialized Stereo-On-1 DAC on LPT port %03xh", lpt_port);
+}
+
+void StereoOn1::ConfigureFilters(const FilterState state)
+{
+	assert(channel);
+	if (state == FilterState::On) {
+		constexpr auto lp_order   = 1;
+		// TBD -- 0.45 sounds a bit too aliasy w/ modplay, maybe more tuning needed
+		const auto lp_cutoff_freq = static_cast<uint16_t>(channel->GetSampleRate() * 0.38);
```

Comment:
I think it's fine. I think I used something between 0.35-0.4 first, which worked fine for lower sample rates, but sounded too muffled at higher rates nearing 44.1k. So i ended up with 0.45. But as the Stereo Covox is limited at 30k, I think this would work better. I approve 😄